### DISCUSSION TITLE
Redefining ctrl+alt+backspace

### DISCUSSION THREAD

**Discussion by vdegenne** [WayfireWM/wayfire#2430] [Q&A]
ctrl+alt+backspace will exit wayfire and go back to the gnome login screen. I searched everywhere on the wayfire UI but I can't find a way to redefine this command.
I would like to keep this keymap but execute `shutdown now` instead, how would I do that?

Thanks

**Comment by ammen99** [comment]
There is no way to do that currently, unfortunately. Please open an issue if there isn't one already (I think there might be)
